### PR TITLE
User can create a flash card

### PR DESCRIPTION
**Pull request by ArthurHwang**
capture3

### CODE REVIEW

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Nice job! I recommend using either a Bootstrap Card to get the border or use the same color as Bootstrap's `form-control` borders.

**Comment by ArthurHwang** [comment]
capture3

File: Form/Form.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+          </div>
+          <div className="form-group">
+            <label>Answer</label>
+            <input type="text" value={props.value.answer} onChange={props.change} name="answer" className="form-control" id="user-answer-input" placeholder="Enter Answer"/>
+          </div>
+          <button type="submit" className="save-btn btn btn-primary">Save</button>
+        </form>
+      </div>
+    </div>
+<<<<<<< HEAD
```
Comment (thebearingedge) [inline Form/Form.js:21]: Looks like there's still some junk left over from your merge conflict.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Looks beautiful! Whenever you update the appearance of the app in a PR, either replace the screenshot/gif or add it to the first comment on the PR.

**Comment by thebearingedge** [comment]
Please reopen when ready.

**Pull request closed without merging**